### PR TITLE
Remove objective c bridging header

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Remove objective c bridging header.
Since frameworks is no longer imported via header, adding necessary `import` in each swift files.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Components/TabBar/TabBarController.swift [hunk 1]
```diff
@@ -168,7 +168,8 @@ final class TabBarController: UIViewController, UIPageViewControllerDelegate, UI
 
         guard toViewController != fromViewController else { return }
         
-        let forward = viewControllers.firstIndex(of: toViewController) > fromViewController.flatMap(viewControllers.firstIndex)
+        //TODO: unwrap
+        let forward = viewControllers.firstIndex(of: toViewController)! > fromViewController.flatMap(viewControllers.firstIndex)!
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Components/TabBar/TabBarController.swift:172]: Is it not possible to fix this now?

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/EphemeralTimeoutFormatter.swift [hunk 2]
```diff
@@ -112,11 +113,11 @@ final class EphemeralTimeoutFormatter {
...
         let date = Date(timeIntervalSinceNow: interval)
 
-        if date < Calendar.current.date(byAdding: .minute, value: 1, to: now) {
+        if date < Calendar.current.date(byAdding: .minute, value: 1, to: now)! {
             return secondsFormatter.string(from: interval)
-        } else if date < Calendar.current.date(byAdding: .hour, value: 1, to: now) {
+        } else if date < Calendar.current.date(byAdding: .hour, value: 1, to: now)! {
             return minuteFormatter.string(from: interval)
-        } else if date < Calendar.current.date(byAdding: .day, value: 1, to: now) {
+        } else if date < Calendar.current.date(byAdding: .day, value: 1, to: now)! {
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/EphemeralTimeoutFormatter.swift:120]: Why do we lose the ability to compare optionals?

File: Wire-iOS/Sources/UserInterface/Components/TabBar/TabBarController.swift (hunk 1 above)
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Components/TabBar/TabBarController.swift]: fixed

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/EphemeralTimeoutFormatter.swift (hunk 2 above)
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/EphemeralTimeoutFormatter.swift:120]: Unknown. I think it was a bug of the compiler to assume return type  as `Date!`. The Swift interface returns an optional.

```
    @available(iOS 8.0, *)
    public func date(byAdding component: Calendar.Component, value: Int, to date: Date, wrappingComponents: Bool = false) -> Date?
```

File: Wire-iOS Tests/MessageDetails/MessageDetailsViewControllerTests.swift [hunk 3]
```diff
@@ -26,7 +26,7 @@ class MessageDetailsViewControllerTests: CoreDataSnapshotTestCase {
     }
     
     // MARK: - Seen
-    
+    ///TODO: status bar?
```
Comment (johnxnguyen) [inline Wire-iOS Tests/MessageDetails/MessageDetailsViewControllerTests.swift:29]: TODO?

Comment (billypchan) [reply Wire-iOS Tests/MessageDetails/MessageDetailsViewControllerTests.swift:29]: The old snapshot test framework sometimes is unstable, I will convert to Swift snapshot framework later.

**Pull request merged**